flash/video/classification/data.py

```diff
@@ -388,3 +348,220 @@ class VideoClassificationData(DataModule):
     """Data module for Video classification tasks."""
 
     input_transform_cls = VideoClassificationInputTransform
+
+    @classmethod
+    def from_files(
+        cls,
+        train_files: Optional[Sequence[str]] = None,
```

Comment:
Should add `input_cls = VideoClassificationFiftyOneInput` there ? Users could modify the dataset class but also, we won't need a registry. We could check all the `from_*` signatures and extract the default class from it associated to the function. Just an idea.

Reply:
Could do, worth exploring when we get to that stage :smiley: